src/encoded/viewconfigs/matrix.py

```diff
@@ -61,7 +62,7 @@ def _construct_query(self):
             del query['query']['query_string']
         else:
             query['query']['query_string']['fields'].extend(
-                ['_all', '*.uuid', '*.md5sum', '*.submitted_file_name']
+                [RESOURCES_INDEX, '*.uuid', '*.md5sum', '*.submitted_file_name']
```

Comment:
These should be field names not indices names?

Reply:
no it's chr indexes that has to be skipped.

Reply:
Yeah but these are `fields` not indices.

Reply:
reverting this change.